src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -917,6 +930,20 @@ public CaseDataBuilder atStateRespondentCounterClaimAfterNotifyDetails() {
 
     public CaseDataBuilder atStateRespondentRespondToClaim(RespondentResponseType respondentResponseType) {
```

Comment:
Can you make method name more descriptive to Spec Fast Track i.e atStateRespondentRespondToClaimSpecFastTrack